cmd/agent/common/common.go

```diff
@@ -42,10 +42,10 @@ var (
 // GetPythonPaths returns the paths (in order of precedence) from where the agent
 // should load python modules and checks
 func GetPythonPaths() []string {
+	// PySitePackages already in sys.path; takes precedence over any additional location
```

Comment:
could we remove the PySitePackages variable altogether? (and keep but reword this helpful comment :)  )